tmail-backend/mailbox/team-mailboxes/src/main/scala/com/linagora/tmail/team/TeamMailbox.scala

```diff
@@ -0,0 +1,26 @@
+package com.linagora.tmail.team
+
+import org.apache.james.core.Username
+import org.apache.james.mailbox.model.{MailboxConstants, MailboxPath}
+
+object Namespace {
+  val TEAM_MAILBOX_NAMESPACE: Namespace = Namespace(MailboxConstants.NAMESPACE_PREFIX_CHAR + "tm")
+}
+
+case class Namespace(value: String) extends AnyVal
+
+case class MailboxName(value: String) extends AnyVal
```

Comment:
`TeamMailboxName` can not contain `.`.

Reply:
The suffix `.Sent` or `.INBOX` is not a part of TeamMailboxName?
if Yes, It will contain `.`

Reply:
No it is not!